Nodejs/Product/Nodejs/ProjectTemplates/AzureExpress4App/app.js

```diff
@@ -57,5 +57,8 @@ app.use(function (err, req, res, next) {
     });
 });
 
+app.set('port', process.env.PORT || 3000);
 
-module.exports = app;
+var server = app.listen(app.get('port'), function () {
+    debug('Express server listening on port ' + server.address().port);
```

Comment:
Did you try running this and seeing if there are errors? I don't see where 'debug' is defined/imported, so I assume this line throws.